scene/property_list_helper.cpp

```diff
@@ -83,7 +91,13 @@ bool PropertyListHelper::is_initialized() const {
 }
 
 void PropertyListHelper::setup_for_instance(const PropertyListHelper &p_base, Object *p_object) {
+	ERR_FAIL_COND(p_base.prefix.is_empty());
+	ERR_FAIL_NULL(p_base.array_length_getter);
+	ERR_FAIL_COND(p_base.property_list.is_empty());
+	ERR_FAIL_NULL(p_object);
```

Comment:
These could be DEV_ASSERTS tbh. No reason to check it for every single instance.